client/rpc-servers/src/lib.rs

```diff
@@ -120,44 +120,44 @@ pub async fn start_http<M: Send + Sync + 'static>(
 	};
 
 	log::info!(
-		"Running JSON-RPC HTTP server: addr={}, allowed origins={:?}",
+		"Running JSON-RPC HTTP server: addr={}, cors={:?}",
 		addr.map_or_else(|_| "unknown".to_string(), |a| a.to_string()),
 		cors
 	);
 
 	Ok(handle)
 }
 
-/// Start WS server listening on given address.
-pub async fn start_ws<M: Send + Sync + 'static>(
+/// Start a JSON-RPC server listening on given address that supports both HTTP and WS.
+pub async fn start<M: Send + Sync + 'static>(
```

Comment:
we can enable folks to inject their own tower middleware now which could useful I reckon but we could do it in another PR